organisations/services.py

```diff
@@ -30,6 +30,7 @@ def get_admin_emails_and_languages():
     return list(
         get_user_model()
         .objects.filter(is_admin=True)
+        .order_by("id")
```

Comment:
This seems like a completely unrelated change, and therefore should be in its own commit IMO.

Reply:
Some reason why the snapshot tests are sometimes rendered in a different order. Thats the reason for this :)

Reply:
Rebase-editted the change to a different commit.